### DISCUSSION TITLE
本地运行site报错

### DISCUSSION THREAD

**Discussion by 2944927590** [antvis/G2#5596] [Q&A] [answered]
- **Node: v18.18.0
- **G2 Version**:v5.1.4
- **Platform**:macOS ventura 13.4
- **Mini Showcase(like screenshots)**:
- 
![image](https://github.com/antvis/G2/assets/11926742/3aea1145-97b4-4338-903b-83e248f7a83c)
![image](https://github.com/antvis/G2/assets/11926742/61c2289d-9d92-4bbf-a279-308283d61b08)
- **CodePen Link**:
本地运行site报错，在site目录下yarn run start 后报错
Error: [BABEL] /analysis/bin/demo/binx-color.ts: The decorators plugin, when .version is '2018-09' or not specified, requires a 'decoratorsBeforeExport' option, whose value must be a boolean. (While processing: "base$3$3$inherits")

**Comment by pearmini** [comment]
看一下 `npm ls @antv/dumi-theme-antv` 的版本

**Comment by 2944927590** [comment]
> 看一下 `npm ls @antv/dumi-theme-antv` 的版本

@5.1.4 /Users/alsy/practice/G2/site
└── @antv/dumi-theme-antv@0.3.5

**Comment by pearmini** [comment, marked as answer]
更新一下 site 文件夹里面的依赖到 `@antv/dumi-theme-antv@0.3.20`
